main.js

```diff
@@ -53,64 +55,42 @@ function generateRandomPalette() {
 	}
 }
 
-function updatePalette(array) {
-	for (var i = 0; i < array.length; i++) {
-		allBoxes[i].style.backgroundColor = array[i];
-		allHexes[i].innerText = array[i];
+function updatePalette(colorValues) {
```

Comment:
Good, much more descriptive than 'array', easier to understand what the code is doing!